src/status.js

```diff
@@ -311,7 +311,7 @@ module.exports = function(AV) {
   /**
    * Create a status query to find someone's published statuses.
    * @since 0.3.0
-   * @param {Object} source The status source.
+   * @param {Object} source The status source, typically an AV.User.
```

Comment:
那么不如把前面的类型也改了？

```suggestion
   * @param {AV.User} source The status source, typically an AV.User.
```

似乎其他的 API 也有类似问题。

Reply:
把 status.js 里的一些 Object 收窄到了 AV.User，status.js 之外的暂时没碰。